src/main/resources/config/config.yaml

```diff
@@ -19,6 +19,11 @@ logging:
       customFields:
         container: "webhooks"
         environment: ${ENVIRONMENT}
+    - type: sentry
+      threshold: ERROR
+      dsn: ${SENTRY_DSN:-https://example.com@dummy/1}
```

Comment:
🤨

Reply:
@alexbishop1 I think this is maybe to stop it throwing if value is not set in the environment. 
I got it from https://github.com/alphagov/pay-ledger/blob/f449a1394fb65be5161206179d60ff19ffc33328/src/main/resources/config/config.yaml#L59 but open to any better ideas?